config/locales/self_service.en.yml

```diff
@@ -43,6 +43,8 @@ en:
       trading_names_filter_placeholder: e.g. firm name
       available_trading_names_filter_placeholder: e.g. firm name
       new_trading_name_link: Add to directory
+      delete_trading_name_button: Remove from Directory
+      delete_trading_name_confirmation: Are you sure you want to remove %{name} from the directory?
```

Comment:
Deleting a trading name cascades to deleting all the advisers associated with it and thus all the questionnaire data related to them. I feel we should make a big thing about this. "Are you sure you want to delete %{name} AND ALL its associated advisers?" or similar. @spencer @neoeno any opinions on this wording-wise?